core/templates/services/utils.service.ts

```diff
@@ -33,11 +33,12 @@ export class UtilsService {
     return true;
   }
 
-  isString(input: any): boolean {
+  isString(input: Object): boolean {
     return (typeof input === 'string' || input instanceof String);
   }
 
-  isEquivalent(a: any, b: any): boolean {
+
+  isEquivalent(a: Object, b: Object): boolean {
```

Comment:
`Object` is also not the correct type here.
@nishantwrp can you confirm how you got these types?

Reply:
To clarify: can we have a more strict type here?

Reply:
As I mentioned earlier, in this function the arguments are meant to be anything. I don't think we can make the types more strict. Please correct me if I'm wrong here. Thanks!

Reply:
Ok, let us add a comment explaining this everywhere.

Reply:
Ditto.

Reply:
Done